angular/core/services/exception_handler.coffee

```diff
@@ -1,11 +1,11 @@
-angular.module('loomioApp').factory '$exceptionHandler', (AppConfig) ->
-  (exception, cause) ->
-    if AppConfig.reportErrors
-      Airbrake.push
-        error:
-          message : exception.toString()
-          stack : exception.stack
-        params:
-          user_id: AppConfig.currentUserId
-
-    console.error "LoomioApp exception:", exception, cause
+# angular.module('loomioApp').factory '$exceptionHandler', (AppConfig) ->
```

Comment:
Have to remember to uncommit this. Actually I might make this so it console logs the exception stack, which would be much helpful.